src/auth/auth.controller.ts

```diff
@@ -115,7 +115,9 @@ export class AuthController {
     res.header(
       'authorization',
       await this.authService.createToken(
-        { user: profile.email },
+        { user: profile.email,
+          exp: Math.round(Date.now() / 1000) + 90
+        },
```

Comment:
`Date.now()` returns the current date as an integer (32 or 64 bits). Are you really sure that you have to use `Math.round`?
```suggestion
        { 
          user: profile.email,
          exp: 90 + (Date.now() / 1000)
        },
```